core/templates/dev/head/pages/base.html

```diff
@@ -122,9 +120,10 @@
   </head>
 
   <body>
-    {% if iframed %}
+    <span ng-if="iframed">
```

Comment:
Should this be div? (Not sure if warnings-and-loader is inline or not, so just checking.)

Reply:
Replacing span with div since there is already a line-break. I was originally going for one-liner span.

Reply:
Sorry, not sure what you mean by that. Can you elaborate?

Note that {{warnings_and_loader()}} will get interpolated into other stuff (grep for warnings_and_loader in the codebase to see what replaces it).

Reply:
I was originally going for `<span ng-if="!iframed" translate="{{warnings_and_loader()}} "></span>` without a line-break ,which is what I usually use span for. However that causes issue with iframe rendering.
Here is the grep for warnings_and_loader
![screen shot 2018-06-10 at 10 17 16 pm](https://user-images.githubusercontent.com/11153258/41209910-665cb430-6cfc-11e8-830e-283a32387faa.png)

Reply:
I don't think warnings_and_loader is something to translate. It's a Jinja macro (see your first search result for details).

Reply:
Understood!